kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py

```diff
@@ -397,16 +493,16 @@
             "firefox",
             "mobile",
             "ios",
-            "firefox-private-network-vpn",
```

Comment:
@akatsoulas I'm wondering why are we removing those products from the `Privacy and security` topic?

Reply:
probably autocompletion. Updating